### PR TITLE
style: Adjust the format of the code and add things relate with policyMap

### PR DESCRIPTION
**Pull request by Edmond-J-A**
Change the 5-space indentation code in model.lua into 4-space indentation,and add policyMap in Assertion and add code relate with it in model.lua (function addDef and function sortPoliciesByPriority)

Signed-off-by: Edmond <edomondja@gmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@rushitote

File: src/model/Model.lua [hunk 1]
```diff
@@ -25,20 +25,20 @@ function Model:new()
     self.__index = self
     self.model = {}
     self.sectionNameMap = {
-         ["r"] = "request_definition",
-         ["p"] = "policy_definition",
-         ["g"] = "role_definition",
-         ["e"] = "policy_effect",
-         ["m"] = "matchers"
+		["r"] = "request_definition",
```
Comment (rushitote) [inline src/model/Model.lua:28]: I see there are tabs used like this in some places, please replace them with normal 4-spaces.

File: src/model/Model.lua [hunk 2]
```diff
@@ -53,48 +53,48 @@ function Model:loadAssertion(model, cfg, sec, key)
...
-
-     if self.model[sec] == nil then
-          self.model[sec] = {}
-     end
+	if value == "" then return false end
 
-     if self.model[sec][key] == nil then
-          self.model[sec][key] = {}
-     end
+	if self.model[sec] == nil then
```
Comment (rushitote) [inline src/model/Model.lua:67]: Please replace these tabs too.

File: src/model/Model.lua [hunk 3]
```diff
@@ -125,9 +125,9 @@ function Model:loadModel(path)
 end
 
 --[[
-     * loadModelFromText loads the model from the text.
-     *
-     * @param text the model text.
+	* loadModelFromText loads the model from the text.
```
Comment (rushitote) [inline src/model/Model.lua:128]: Here too.

**Review by rushitote — CHANGES_REQUESTED** [review]

File: src/model/Model.lua (hunk 3 above)
Comment (Edmond-J-A) [reply src/model/Model.lua]: Got it，tabs have been changed

**Pull request closed without merging**